codelists/+partyRole.csv

```diff
@@ -0,0 +1,2 @@
+Code,Title,Description
+wholesaleBuyer,Wholesale buyer,"A buyer (e.g. a central purchasing body) buying goods, works or services intended for other buyers. Organisations with this role should not have the buyer role."
```

Comment:
Let's omit the second sentence for now. I'm hoping Jáchym can clarify why he thought it was important to include in https://github.com/open-contracting/standard/issues/1180

```suggestion
wholesaleBuyer,Wholesale buyer,"A buyer (e.g. a central purchasing body) buying goods, services or works intended for other buyers."
```